action.php

```diff
@@ -486,10 +494,21 @@ protected function generatePDF($cachefile, $event) {
         // loop over all pages
         $counter = 0;
         $no_pages = count($this->list);
+        $hierarchy = $INPUT->bool('book_hierarchy');
```

Comment:
The default configuration settings should be used. Please not only on the fly url parameters.